apps/tailwind-components/components/input/CheckboxGroup.vue

```diff
@@ -30,16 +42,17 @@
         </span>
       </InputLabel>
     </div>
-    <div class="mt-2" v-if="showClearButton">
-      <button
-        type="reset"
-        :id="`${id}-checkbox-group-clear`"
-        :form="`${id}-checkbox-group`"
-        @click.prevent="resetModelValue"
-      >
-        Clear
-      </button>
-    </div>
+    <ButtonText
+      v-if="showClearButton"
+      type="reset"
+      :id="`${id}-checkbox-group-clear`"
+      class="mt-2 ml-3"
+      :form="`${id}-checkbox-group`"
+      @click.prevent="resetModelValue"
+      :disabled="disabled || null"
+    >
```

Comment:
ButtonText only has one property (icon), why does disabled work here? If these properties are transitive, I would really like to make them explicit on the component, because now it is very confusing and a setup for bugs.

Reply:
This is a good point. This is something we can revisit as we refactor the button components.